code/modules/mob/living/carbon/say.dm

```diff
@@ -46,4 +46,14 @@
 	for(var/T in get_traumas())
 		var/datum/brain_trauma/trauma = T
 		message = trauma.on_hear(message, speaker, message_language, raw_message, radio_freq)
+
+	if (src.mind.has_antag_datum(/datum/antagonist/traitor))
+		for (var/codeword in GLOB.syndicate_code_phrase)
+			var/regex/codeword_match = new("(" + codeword + ")", "ig")
```

Comment:
string concat is really slow compared to format. This should do the same thing but faster.
```suggestion
			var/regex/codeword_match = new("([codeword])", "ig")
```

Reply:
Ah fair, I did this initially but was worried since [ is also valid regex, so wasn't sure that worked.

Reply:
formats are done before any kind of parameter passing, don't worry about it.